benchmarks/local-neutral/reports/http-runtime-hotpath-pass3-2026-05-14.md

```
@@ -0,0 +1,131 @@
+| --- | --- | --- | --- |
+| POST confirmation | Disable pass-2 text response path and compare five repeats | Regression did not repeat | Keep pass-2 fast path |
+| Route/context | Generated route slot helper over JS context/private values | Correct after fixes, but RPS did not beat pass 2 and memory roughly doubled | Reverted |
+| Body/validation | Request-body JSON binding without public `ctx.body` facade | `post-json-validated` improved more than 10% in focused runs; memory neutral-to-better | Kept |
+
+## Validation
+
+| Command | Result |
+| --- | --- |
+| `tools/windows/dev.ps1 configure -Preset windows-relwithdebinfo -EnableV8` | PASS |
+| `tools/windows/dev.ps1 build -Preset windows-relwithdebinfo` | PASS |
+| `ctest --test-dir build/windows-relwithdebinfo -C RelWithDebInfo -R "^(core\.request_validation|core\.http\.dispatch|engine\.v8\.smoke|conformance\.v8\.runtime_bridge|http\.dispatch\.execution|conformance\.v8\.http_dispatch_execution|benchmarks\.local_neutral\.contract)$" --output-on-failure` | PASS |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 8; Too many cells, extra data will be missing

(MD056, table-column-count)